src/App.js

```diff
@@ -55,10 +55,6 @@ function App() {
         <Route exact path="/orders">
           <Orders />
         </Route>
-        <Route exact path="/reviews">
-          <Reviews />
-        </Route>
```

Comment:
I don't believe this route was being used. Since the `<Reviews>` component specifically is written to be reviews for a specific product.